samples/apply-class-breaks-renderer-to-sublayer/src/main/java/com/esri/arcgismaps/sample/applyclassbreaksrenderertosublayer/screens/ApplyClassBreaksRendererToSublayerScreen.kt

```diff
@@ -0,0 +1,89 @@
+/* Copyright 2025 Esri
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package com.esri.arcgismaps.sample.applyclassbreaksrenderertosublayer.screens
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.Button
+import androidx.compose.material3.Scaffold
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.unit.dp
+import androidx.lifecycle.viewmodel.compose.viewModel
+import com.arcgismaps.toolkit.geoviewcompose.MapView
+import com.esri.arcgismaps.sample.applyclassbreaksrenderertosublayer.components.ApplyClassBreaksRendererToSublayerViewModel
+import com.esri.arcgismaps.sample.sampleslib.components.MessageDialog
+import com.esri.arcgismaps.sample.sampleslib.components.SampleTopAppBar
+
+/**
+ * Main screen layout for the sample app
+ */
+@Composable
+fun ApplyClassBreaksRendererToSublayerScreen(sampleName: String) {
+
+    val mapViewModel: ApplyClassBreaksRendererToSublayerViewModel = viewModel()
+
+    val isSublayerReady = mapViewModel.isSublayerReady
+    val classBreaksRendererIsApplied = mapViewModel.classBreaksRendererIsApplied
+
+    Scaffold(
+        topBar = { SampleTopAppBar(title = sampleName) },
+        content = {
+            Column(
+                modifier = androidx.compose.ui.Modifier
```

Comment:
Add imports, for modifiers below as well: 
```suggestion
                modifier = Modifier
```